qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -107,6 +107,34 @@
   </screenshots>
 
   <releases>
+   <release version="2022.11.25-2" date="2022-11-25">
```

Comment:
We can tag the release today )

Reply:
OK, changed it to `2022-11-25`, the suffix is still `-2`, I have never followed how that is determined.

Reply:
It's the number of commits in the day when the release/tag is created. It is generated/calculated by tools/unix/version.sh